### PR TITLE
Modified the bulk transaction label

### LINKED ISSUE #3975: Forking: use new bulk tx label
Add number of transactions to the message. Update this component to new look and location in modals. Wrap `DismissableNotice` component
`<BulkTxLabel buttonName={'Claim All'} count={props.submitAllTxCount} needsApproval={false} />`

3 places current used:
 * update claim modal
 * update unsigned orders
 * create market 

https://www.figma.com/file/aAzKHh4cA6OT2t7WFv2BQ7fB/Reporting-and-Disputing?node-id=2157%3A283

### PR DESCRIPTION
**Pull request by bconfortin**
Closes https://github.com/AugurProject/augur/issues/3975

Tested on multiple resolutions

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🍦

 This link will always point to this PR's latest changes: https://peek.run/7g6hvz0

File: packages/augur-ui/src/modules/common/labels.styles.less [hunk 1]
```diff
@@ -704,9 +704,11 @@ div.LinearPropertyViewTransaction {
 }
 
 .BulkTxLabel {
-  .mono-9;
+  margin: 0 @size-24 @size-24;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/labels.styles.less:707]: how does the font size get set? don't want to inherit it.

Comment (bconfortin) [reply packages/augur-ui/src/modules/common/labels.styles.less:707]: Via `DismissableNotice` css.

`.mono-9` was the old font for when there was just text, without the white/blue label

Comment (bthaile) [reply packages/augur-ui/src/modules/common/labels.styles.less:707]: ok, I rem now, works for me.

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**